scripts/attendance.py

```diff
@@ -0,0 +1,167 @@
+        :param available_working_days:
+        :param present_days:
+        :return:
+        """
+        if average_attendace == '09:00:00' or\
+            average_attendace.startswith('09:00:') or\
+                average_attendace.startswith('9:00:'):
+            return 9, 00, available_working_days
+        avrage_working_time_list = average_attendace.split(":")
+        average_attendace_hours = int(avrage_working_time_list[0])
+        average_attendace_minutes = int(avrage_working_time_list[1])
+        if average_attendace_hours < DAILY_WORKING_HOURS:
+            tt = timedelta(hours=DAILY_WORKING_HOURS) - \
+                 (timedelta(hours=average_attendace_hours) +
+                  timedelta(minutes=average_attendace_minutes))
+            res = timedelta(hours=9) +\
+                timedelta(minutes=self.time_to_minutes(present_days*tt)) /\
+                (available_working_days)
+            result_list = str(res).split(":")
+            hours, minutes = result_list[0], result_list[1][:2]
+            return hours, minutes, available_working_days
+        else:
+            tt = (timedelta(hours=average_attendace_hours) +
+                  timedelta(minutes=int(average_attendace_minutes))) - \
+                 timedelta(hours=DAILY_WORKING_HOURS)
+            mins = self.time_to_minutes(tt)
+            td = timedelta(minutes=mins * present_days)
+            days, hours, minutes = td.days, td.seconds // 3600, td.seconds //\
+                60 % 60
+            extra_work = timedelta(hours=days * 24) + timedelta(hours=hours) +\
+                timedelta(minutes=minutes)
+            retData = (timedelta(days=available_working_days) - extra_work) / \
+                timedelta(hours=9)
+            result_list = str(retData).split(".")
+            hours, minutes = result_list[0], result_list[1][:2]
+            return hours, minutes, available_working_days
+
+
+def is_validate_time_format(average_attendace):
+    """
+    This will check is user entered time is valid or not
+    :param average_attendace:
+    :return:
+    """
+    timeformat = "%H:%M:%S"
+    try:
+        valid_time = datetime.strptime(average_attendace, timeformat)
+    except ValueError:
+        print("Please enter average present time in hh:mm:ss format")
+        attendance()
+
+
+def is_valid_integer(number, error_message):
+    """
+    This will check is user entered value is valid integer or not
+    :param number:
+    :param error_message:
+    :return:
+    """
+    try:
+        valid_number = input(number)
+    except ValueError:
+        print(error_message)
+        attendance()
+
+
+def attendance():
+    attendance_management = AttendanceManagement()
+    average_attendace = input("Entaer your average attendance in 'hh:mm:ss'"
+                              " format. E.g. 8:00:00  ")
+    is_validate_time_format(average_attendace)
+
+    present_days = int(input("Enter your present days. E.g. 19  "))
```

Comment:
Best practice is always use some command line interface eg: https://docs.python.org/3/library/argparse.html